addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3338,7 +3338,222 @@ msgctxt "#802"
 msgid "%s of %s available"
 msgstr ""
 
-#empty strings from id 803 to 849
+#. Label of spinner control for choosing the timer type (one time, repeating, ...)
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#803"
+msgid "Type"
+msgstr ""
+
+#. Label of 'Guide data title search' edit field
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#804"
+msgid "Search guide for"
+msgstr ""
+
+#. Help of 'Guide data title search' edit field
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#805"
+msgid "Enter the search string to use to lookup matching events in the guide"
+msgstr ""
+
+#. Label of "Fulltext search" edit field
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#806"
+msgid "Fulltext search (or title search only)"
+msgstr ""
+
+#. special weekdays representation value for a timer schedule
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#807"
+msgid "Any day"
+msgstr ""
+
+#. special weekdays representation for a repeating timer
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#808"
+msgid "Every day"
+msgstr ""
+
+#. special channel selection value for a repeating timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#809"
+msgid "Any channel"
+msgstr ""
+
+#. Label of "Any time" radio button
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#810"
+msgid "Any time"
+msgstr ""
+
+#. Label of "Day" edit field
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#811"
+msgid "Day"
+msgstr ""
+
+#. Label of only new episodes radio button (if user does not want to record the same episode multiple times)
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#812"
+msgid "Only new episodes"
+msgstr ""
+
+#. Label of pre-record spinner
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#813"
+msgid "Start padding time"
+msgstr ""
+
+#. Label of post-record spinner
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#814"
+msgid "End padding time"
+msgstr ""
+
+#empty strings from id 815 to 819
+
+#. Representation of a one-time timer.
+#: xbmc/pvr/addons/PVRClient.cpp
+msgctxt "#820"
+msgid "One time"
+msgstr ""
+
+#. Representation of an epg-based one-time timer.
+#: xbmc/pvr/addons/PVRClient.cpp
+msgctxt "#821"
+msgid "One time (guide-based)"
+msgstr ""
+
+#. Representation of a timer schedule
+#: xbmc/pvr/addons/PVRClient.cpp
+msgctxt "#822"
+msgid "Repeating"
+msgstr ""
+
+#. Representation of an epg-based timer schedule.
+#: xbmc/pvr/addons/PVRClient.cpp
+msgctxt "#823"
+msgid "Repeating (guide-based)"
+msgstr ""
+
+#empty string with id 824
+
+#. Representation of PVR timer state "disabled"
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#825"
+msgid "Disabled"
```

Comment:
me again, I really don't want to bikeshed, but even translators already complained about all the duplicates we're having, and there are already quite some "disabled" labels available. Mind checking if they can be reused? Like 1223, 13106, 13113, 13131, 24023. If the context they're used in doesn't match, keep yours, but I think at least the label used in settings.xml could match.

All this duplicates need to be cleaned up at some point, so the more duplicates we can avoid now the better.